VSharp.ML.AIAgent/ml/common_model/utils.py

```diff
@@ -58,23 +62,21 @@ def csv2best_models():
                 best_model = max(models, key=(lambda m: m[1][map_name]))
                 best_model_name = best_model[0]
                 best_model_score = best_model[1]
```

Comment:
называется "variable unpacking"
```suggestion
                best_model_name, best_model_score = best_model[0], best_model[1]
```